### PR TITLE
Add ability to output csv to mti_cmap_data tool.

### PR DESCRIPTION
**Pull request by dougfelt**
Because Google Spreadsheets still (!) does not provide an option to prevent
autoconversion of text to numbers, this puts 'U+' in front of each cell
so that Google Spreadsheets will leave it alone.  Then afterwards you can
mark all columns as text only in Google Spreadsheets, and replace all to remove
the leading U+.  Otherwise some of the hex is interpreted as decimal numbers,
losing the leading zeros, and some as text.  Urg.

### CODE REVIEW

File: nototools/mti_cmap_data.py [hunk 1]
```diff
@@ -81,24 +81,92 @@ def cmap_data_from_csv_file(csvfile):
   return cmap_data_from_csv(csvdata, csvfile)
 
 
+def csv_to_xml(csv_file, xml_file, scripts):
+  cmapdata = cmap_data_from_csv_file(csv_file)
+  if scripts:
+    print >> sys.stderr, 'not filtering by scripts when converting csv to xml'
```
Comment (jamesgk) [inline nototools/mti_cmap_data.py]: It was a little confusing to me what this message was for at first. I might make it more explicit, like "not using 'scripts' argument when..."

**Comment by jamesgk** [comment]
LGTM

File: nototools/mti_cmap_data.py (hunk 1 above)
Comment (dougfelt) [inline nototools/mti_cmap_data.py]: Yeah, I'll either implement this or remove the argument.

**Comment by dougfelt** [comment]
There's significant additional changes, can you look at my last commit?

**Comment by jamesgk** [comment]
Looks good

**Pull request merged**